Bundle size

hypervillain opened the pull request:
Following #27, I rewrote the library with bundle size in mind.

Changes:

- Removed polyfills, updated README
- Use of `transform-react-remove-prop-types` for production build
- Added bundle analyzer + webpack command
- Removed utility not critical to the application (+ calls to `react-is`)

Test:

your existing test websites should work

StarpTech commented:
What's the browser support in general?
Instead to require polyfills I would rewrite the code or transpile it down to es5.

MarcMcIntosh commented on src/richtext.js:
```diff
@@ -61,8 +55,8 @@ function serializeSpan(content) {
       } else {
-        const brIndex = (acc.length + 1)/2 - 1;
-        const br = React.createElement('br', propsWithUniqueKey({}, brIndex));
+        const brIndex = (acc.length + 1) / 2 - 1;
```
@hypervillain  small note it's the same formula but ((acc.length + 1) / 2) - 1 is easier to read

hypervillain commented:
@StarpTech I removed the polyfills and added an entry in the README that should list which one may have to be added. And because we use `@babel/preset-env` the code should be transpiled down correctly. Do you see other opitmizations we could add?
